fact_extractor/plugins/unpacking/microtik/install.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+sudo -EH pip3 install --upgrade npkPy
```

Comment:
I recommend to inform the installer if installation fails
`sudo -EH pip3 install --upgrade npkPy || exit 1`